### ISSUE TITLE
Provide Known Answer Tests

### ISSUE THREAD

**Issue by mberry** [pegnet/LXR256#56]
Hashes and ciphers usually have known answer tests associated with the reference implementation.

This provides a canonical digest for anyone writing their own implementation of LXR Hash to check against.

**Comment by WhoSoup** [comment]
at the moment, we have: <https://github.com/pegnet/LXRHash/blob/ca3b8b65a729b40fb81a931bf02746a1701bdb96/lxrhash_test.go#L46-L71>

Is that good enough or are we talking about a list of hundreds/thousands of them?

**Comment by mberry** [comment]
Yeah that's great, a few is fine. Some hashes/ciphers seem to go overboard testing 1000's of edgecases, doubt that's of any use here. 
 
 Wrote a LXR implementation and trying to keep it simple wrt keeping up with any algorithm changes. Hopefully a KAT file or some notes in the readme can help anyone else doing the same.
